torchtext/data/utils.py

```diff
@@ -110,6 +110,8 @@ def get_tokenizer(tokenizer, language='en'):
         return tokenizer
 
     if tokenizer == "spacy":
+        if language == 'en':
+            raise RuntimeError("The en package has been deprecated in Spacy 3.0 release. Please switch to en_core_web_sm.")
```

Comment:
I'd make explicit reference to the "language" keyword argument used above. A user might not know that this translate into a spacy package and what to change about their code to fix this issue.

Reply:
Instead of throwing an error we can detect this and add a warning that we're using "en_core_web_sm" instead. The same applies to "de". That way we also maintain BC compatibility and users don't need to update their API.